packages/eas-cli/schema/metadata-0.json

```diff
@@ -0,0 +1,928 @@
+            "type": "string",
+            "maxLength": 30,
+            "meta": {
+              "storeInfo": "The subtitle will be reviewed before it is made available on the App Store."
+            }
+          },
+          "description": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true
+            },
+            "maxLength": 4000,
+            "description": "Main description of what the app does",
+            "type": "string"
+          },
+          "promoText": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true,
+              "aso": false
+            },
+            "maxLength": 170,
+            "description": "Short tagline for the app",
+            "type": "string"
+          },
+          "keywords": {
+            "meta": {
+              "versioned": true
+            },
+            "description": "List of keywords to help users find the app in the App Store",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+              "maxLength": 100,
+              "type": "string"
+            }
+          },
+          "privacyPolicyText": {
+            "type": "string",
+            "description": "Privacy policy for Apple TV"
+          },
+          "whatsNew": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true
+            },
+            "maxLength": 4000,
+            "description": "Changes since the last public version",
+            "type": "string"
+          },
+          "marketingUrl": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true
+            },
+            "description": "URL to the app marketing page",
+            "type": "string"
+          },
+          "supportUrl": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true
+            },
+            "description": "URL to the app support page",
+            "type": "string"
+          },
+          "privacyPolicyUrl": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true
+            },
+            "maxLength": 255,
+            "description": "A URL that links to your privacy policy. A privacy policy is required for all apps.",
+            "type": "string"
+          },
+          "privacyChoiceUrl": {
+            "meta": {
+              "versioned": true,
+              "liveEdits": true,
+              "optional": true
+            },
+            "maxLength": 255,
+            "description": "A URL where users can modify and delete the data collected from the app, or decide how their data is used and shared.",
+            "type": "string"
+          }
+        }
+      },
+      "AppleAdvisory": {
+        "type": "object",
+        "additionalProperties": false,
+        "defaultSnippets": [
```

Comment:
Probably safe to delete these default snippets for now, can add back later when we update to match the latest spec.